scripts/uboot_helper

```diff
@@ -199,28 +199,20 @@ devices = \
   },
   'NXP': {
     'iMX6': {
-      'cubox-dl': {
-        'dtb': 'imx6dl-cubox-i.dtb',
+      'generic': {
```

Comment:
Please remove the generic option. This isn't relevant to this PR.

Reply:
OK, generic option removed